apis_ontology/serializers.py

```diff
@@ -301,7 +301,7 @@ def to_representation(self, instance):
             (
                 appellation_uri,
                 RDFS.label,
-                Literal(f"{instance.name}"),
+                Literal(instance),
```

Comment:
as described in the commit. either this needs ti be fixed, or this pr can wait until the group serializer in main is merged and then remove the institution serializer all together.